src/annotator/sidebar.js

```diff
@@ -40,6 +40,7 @@ function createSidebarIframe(config) {
 
   // Enable media in annotations to be shown fullscreen
   sidebarFrame.setAttribute('allowfullscreen', '');
+  sidebarFrame.setAttribute('allow', 'clipboard-write');
```

Comment:
Drat: It seems that it doesn't work in the LMS context in Chrome. It does work in the LMS context in Safari and Firefox.

In the LMS context the Hypothesis client is loaded inside an iframe which is created by the LMS frontend. The LMS frontend in turn may either be 1) the top level tab, or 2) it may be contained inside an iframe created by the LMS itself. We can make the new API work in case (1) but not case (2).

I tested that adding `allow="clipboard-write"` to the iframe created by the `BasicLtiLaunchApp` component in the LMS frontend resolves the problem for case (1). I haven't figured out yet if we can get it to work for case (2) at all.

Reply:
Does it fallback to the legacy version?